### PR TITLE
Add expiry value

### PR DESCRIPTION
**Pull request by zlu-DFE**

### CODE REVIEW

**Review by gcompton-DFE — APPROVED** [review]
Reviewed

File: DFC.Api.JobProfiles.ProfileServices/ProfileDataService.cs [hunk 1]
```diff
@@ -24,6 +24,7 @@ namespace DFC.Api.JobProfiles.ProfileServices
 {
     public class ProfileDataService : IProfileDataService
     {
+        private const double expiry = 24;
```
Comment (abol-edu) [inline DFC.Api.JobProfiles.ProfileServices/ProfileDataService.cs:27]: What does 24 mean? Seconds, minutes, hours, days? Something unrelated to time?

Comment (ZhaomingLu2009) [reply DFC.Api.JobProfiles.ProfileServices/ProfileDataService.cs:27]: It's 24 hours

Comment (abol-edu) [reply DFC.Api.JobProfiles.ProfileServices/ProfileDataService.cs:27]: Could this be made more explicit? For example by renaming the variable to `expiryTimeInHours`.

**Pull request merged**